homeassistant/components/climate/ecobee.py

```diff
@@ -382,13 +383,3 @@ def hold_preference(self):
             return default
         else:
             return 'nextTransition'
-
-    # Sleep mode isn't used in UI yet:
```

Comment:
I think you are confusing hold with schedule, see more detail in my comment to #5590.